source/numerics.tex

```diff
@@ -9472,7 +9472,7 @@
 if \tcode{X} cannot be converted to \tcode{int}
 by integral promotion\iref{conv.prom}, the program is ill-formed.
 \begin{note}
-Arguments that can be promoted to \tcode{int} are permitted for compatibility with C.
+Arguments that can be promoted to \tcode{int} are supported for compatibility with C.
```

Comment:
Alternatively (but I don't think it's necessarily better):

```suggestion
Arguments that can be promoted to \tcode{int} provide compatibility with C.
```

Reply:
Or "allowing arguments that can be promoted to `int` provide compatibility with C".

Reply:
Yes to the latter (it's not the arguments that provide compatibility, but this feature as a whole).